### DISCUSSION TITLE
Supernets

### DISCUSSION THREAD

**Discussion by kirillt** [0xPolygon/polygon-edge#532] [General]
Is it something what can be tried-out already or only upcoming idea?
https://blog.polygon.technology/introducing-polygon-supernets-powered-by-polygon-edge-100m-ecosystem-fund/

**Comment by lazartravica** [comment]
Hello @kirillt , 

As Polygon Supernets is a program powered by the technology of Polygon Edge, you would need to apply for it here to enter the process of receiving the grant:

https://docs.google.com/forms/d/11abPLDEEASA8qdG5MuaO9KxHsLapKywac2YNIoD8iyE/viewform

You can also read more about Supernets here: https://polygon.technology/solutions/polygon-supernets

Thanks for showing interest 🤘

**Comment by kirillt** [comment]
Hi @lazartravica,

Thank you for the links. Am I right that supernets are not implemented yet, but me and other developers are encouraged to try ourselves in creating it in scope of a grant?

> You can also read more about Supernets here: https://polygon.technology/solutions/polygon-supernets

This links leads to blank page on my devices (Firefox, Chrome on laptop, Firefox on Android).

**Comment by laviniat1996** [comment]
Hi @kirillt ,

Please check out this **link**, it should work: https://polygon.technology/polygon-edge-supernets/

As for the implementation of Supernets, you are right, they are not implemented yet. A **big planning session for Supernets** will take place on May 30th: we are about to start with **Supernets Testnets**, and then we'll slowly move forward with the validators.

**Comment by kirillt** [comment]
@laviniat1996 thank you for clarification. The link works now as well.

**Reply by laviniat1996** [reply]
@kirillt You are very welcome. Anytime you have a question/ curiosity, do not hesitate to ask us.
